elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -141,6 +141,13 @@ abstract class RestlasticSearchClient(endpointProvider: EndpointProvider, signer
     }
   }
 
+  def sampleAggregation(index: Index, tpe: Type, query: AggregationQuery): Future[BucketAggregationResultBody] = {
```

Comment:
Don't we want to support sample aggregations for multiple indices in the short/long term world?

Reply:
Btw. we should ensure that the current Magellan implementation works with two indices. (I did the first check, but I will re-ensure)

Reply:
Magellan uses a different way to run sampler aggregations and it will work with more indices.